.travis.yml

```diff
@@ -7,33 +7,59 @@ notifications:
     - michele@openquake.org
     - daniele@openquake.org
 
-python:
- - "3.5"
-
 services:
  - rabbitmq
 
 cache:
   pip: true
   directories:
-  - wheels
+  - wheels27
+  - wheels35
 
 env:
   - OQ_DISTRIBUTE=celery
 
 jobs:
   include:
   - stage: tests
-    env: ENGINE
+    env:
+      - OQ_DISTRIBUTE=celery
+      - PYTHON=2.7
+    python: "2.7"
     script:
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.engine
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.server
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.calculators
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.risklib
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.commonlib
-        - OQ_DISTRIBUTE=celery nosetests --with-doctest -xvs -a'!slow' openquake.commands
+        - nosetests --with-doctest -xvs -a'!slow' openquake.engine
+        - nosetests --with-doctest -xvs -a'!slow' openquake.server
+        - nosetests --with-doctest -xvs -a'!slow' openquake.calculators
+        - nosetests --with-doctest -xvs -a'!slow' openquake.risklib
+        - nosetests --with-doctest -xvs -a'!slow' openquake.commonlib
```

Comment:
I would try to do this in a single line
```
 nosetests --with-doctest -xvs -a openquake
```
also including the slow tests, so we do not forget the hmtk or directories that may be added in the future

Reply:
Agreed